src/metabase/models/revision/diff.clj

```diff
@@ -119,14 +119,13 @@
   (when-let [[before after] (data/diff o1 o2)]
     (let [ks         (keys (or after before))
           model-name (model-str->i18n-str model)]
-      (filter identity
-              (map-indexed (fn [i k]
-                             (diff-string k (k before) (k after)
-                                          (if (zero? i) (deferred-tru "this {0}" model-name) (deferred-tru "it"))))
-                           ks)))))
-:used-underscored-binding
-
-(let [x 1 y 2]
-  (clojure.core.match/match [:name x y]
-    [:name _ _]
-    "Ok"))
+      (loop [ks               ks
```

Comment:
we had a failing e2e test because of this https://github.com/metabase/metabase/actions/runs/6714770177/job/18249901919?pr=34947